engine/src/main/java/com/datatorrent/stram/debug/StdOutErrLog.java

```diff
@@ -54,6 +56,7 @@ public static void tieSystemOutAndErrToLog()
 
     System.setOut(createLoggingProxy(System.out));
     System.setErr(createLoggingProxy(System.err));
+    LoggerUtil.addAppender("apex");
```

Comment:
What about DT_LOG_APPENDER that is being configured above. Can it be repurposed for this? Also, looks like createLoggingProxy is a pass through and system out and err are going to out and err as usual. Are those proxies needed?